features/net/FEATURE_IPV4/TESTS/mbedmicro-net/nist_internet_time_service/main.cpp

```diff
@@ -8,55 +8,64 @@
 #include "greentea-client/test_env.h"
 
 namespace {
-    //const char *HTTP_SERVER_NAME = "utcnist.colorado.edu";
     const char *HTTP_SERVER_NAME = "pool.ntp.org";
     const int HTTP_SERVER_PORT = 123;
```

Comment:
@jeromecoutant Adding a command line option for that would be a useful, albeit non-trivial change. We'd have to discuss that a bit more before moving forward on it.

The quickest way to test locally would be to just change the address of the NTP server in your code, though I realize that's not a long-term solution.